File: src/main/java/com/giant/common/api/doc/success/SuccessResponseDoc.java
package com.giant.common.api.doc.success;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Getter;

@Schema(name = "SuccessResponse")
@Getter
public abstract class SuccessResponseDoc {
    @Schema(example = "SU")
    private final String code = null;
    @Schema(example = "요청이 성공적으로 처리되었습니다.")
    private final String message = null;
}

File: src/main/kotlin/com/giant/common/api/doc/success/SuccessResponseDoc.kt
package com.giant.common.api.doc.success

import io.swagger.v3.oas.annotations.media.Schema

@Schema(name = "SuccessResponse")
abstract class SuccessResponseDoc {
    @get:Schema(example = "SU")
    val code: String? = null
    @get:Schema(example = "요청이 성공적으로 처리되었습니다.")
    val message: String? = null
}